recipe/migrations/osx_arm64.txt

```diff
@@ -138,3 +138,7 @@ openpmd-api
 jemalloc
 xsimd
 line_profiler
+great-expectations
+rejected
+xgboost
```

Comment:
This needs R packages, and we should add that only after adding the mini-migrator.

Reply:
Only discovered that once the migrator has started. I'll have a look at the mini-migrator on Wednesday and will fix the open PRs.